lib/core/utils/valid-langs.js

```diff
@@ -1,8153 +1,105 @@
+
+/**
+ * Determine if a string is a valid language code
+ * @method isValidLang
+ * @memberof axe.utils
+ * @param {String} lang String to test if a valid language code
+ * @returns {Boolean}
+ */
+function isValidLang(lang) {
+	let array = langs;
+
+	// padEnd is not supported in IE11
+	while (lang.length < 3) {
+		lang += '`';
+	}
+	for (let i = 0; i <= lang.length - 1; i++) {
+		const index = lang.charCodeAt(i) - 96;
```

Comment:
Just curious.. what does the `96` here refer to?

--------- 

(awesome PR. so pumped for this.)

I had also found this regex-based approach last week: https://github.com/SafetyCulture/bcp47/blob/develop/src/index.js  but it's more of a "does it look roughly like a bcp47 code" as opposed to being specific.

Reply:
the 96 gets a 1 index on the letter 'a' and a 0 index on letter '`'